### PR TITLE
Documentation: Search Feature

### LINKED ISSUE #76: Week 9: Update UG - Search Feature

### LINKED ISSUE #81: Week 9: Update DG - Implementation: Search

### PR DESCRIPTION
**Pull request by florentianayuwono**
Complete #76 and #81

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -161,21 +161,23 @@ Example:
 
 ###  Search for a Contact: `search`
 
-Search for contacts using tags or other information.
+Search for contacts using partial information.
 
-Format: `search [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…`
+Format: `search [CONDITION] [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…`
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md:166]: May I know what are the `CONDITION` there are?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -287,7 +289,7 @@ _Details coming soon ..._
 | **Customise**  | `customise hide [p/] [e/] [a/] [t/]` <br> `customise show [p/] [e/] [a/] [t/]` <br> `customise order [p/] [e/] [a/] [t/]` <br> e.g, `customise order a/ e/ p/` |
 | **Delete**     | `delete INDEX`<br> e.g., `delete 3`                                                                                                                            |
 | **Edit**       | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS]`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                              |
-| **Search**     | `search [n/NAME] [p/PHONE_NUMBER] ...`<br> e.g., `seach n/John Doe t/cs2103t`                                                                                  |
+| **Search**     | `search [CONDITION] [n/NAME] [p/PHONE_NUMBER]…​`<br> e.g., `seach n/John Doe t/cs2103t`                                                                        |
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md]: Maybe u could use `CONDITION` in the example?

**Review by ChongCheeKaiClarence — COMMENTED** [review]
Looks Great! Just need some edits and explanation.

File: docs/UserGuide.md (hunk 1 above)
Comment (florentianayuwono) [reply docs/UserGuide.md:166]: the `and` and `or`

File: docs/UserGuide.md (hunk 2 above)
Comment (florentianayuwono) [reply docs/UserGuide.md]: okay!

**Review by Eugene-Ong-W-X — COMMENTED** [review]
LGTM

**Pull request merged**